ledger/src/blockstore_processor.rs

```diff
@@ -1339,11 +1339,8 @@ impl ThreadExecuteTimings {
 
     pub fn accumulate(&mut self, other: &ThreadExecuteTimings) {
         self.execute_timings.accumulate(&other.execute_timings);
-        saturating_add_assign!(self.total_thread_us, other.total_thread_us);
-        saturating_add_assign!(
-            self.total_transactions_executed,
-            other.total_transactions_executed
-        );
+        self.total_thread_us += other.total_thread_us;
+        self.total_transactions_executed += other.total_transactions_executed;
```

Comment:
This isn't saturating anymore.

Reply:
It is still saturating math as the types were declared `Saturating` above:
```rust
    pub total_thread_us: Saturating<u64>,
    pub total_transactions_executed: Saturating<u64>,
```
https://doc.rust-lang.org/beta/std/num/struct.Saturating.html